addon/templates/components/frost-expand.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="frost-expand-label"  data-test={{hook (concat hook '-label')}} onClick={{action '_onClick'}}>
+  {{frost-icon
+    class='frost-expand-label-icon'
```

Comment:
Don't see how the `chevron` is different from the `expand-collapse` one other than the `expand-collapse` one is bigger